app-emulation/fuse/fuse-1.5.2.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Free Unix Spectrum Emulator by Philip Kendall"
+HOMEPAGE="http://fuse-emulator.sourceforge.net"
+SRC_URI="mirror://sourceforge/fuse-emulator/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="alsa ao backend-fbcon +backend-gtk backend-sdl backend-svga backend-X gpm joystick memlimit png xml"
+
+# Only one UI back-end can be enabled at a time
+REQUIRED_USE="^^ ( backend-fbcon backend-gtk backend-sdl backend-svga backend-X )"
+
+RDEPEND=">=app-emulation/libspectrum-1.4.1
+	alsa? ( media-libs/alsa-lib )
+	ao? ( media-libs/libao )
+	backend-gtk? ( x11-libs/gtk+:3 )
+	backend-sdl? ( media-libs/libsdl )
+	backend-svga? ( media-libs/svgalib )
+	backend-X? ( x11-libs/libX11 x11-libs/libXext )
+	dev-libs/glib:2
+	gpm? ( sys-libs/gpm )
+	joystick? ( media-libs/libjsw )
+	png? ( media-libs/libpng:0= sys-libs/zlib )
+	xml? ( dev-libs/libxml2:2 )"
+DEPEND="${RDEPEND}
+	backend-fbcon? ( virtual/linux-sources )
+	dev-lang/perl
+	virtual/pkgconfig"
+
+DOCS=( AUTHORS ChangeLog README THANKS )
+
+src_configure() {
+	local myconf=(
+		--without-win32
+		$(use_with alsa)
+		$(use_with ao libao)
+		$(use_with gpm)
+		$(use_with joystick)
+		$(use_enable joystick ui-joystick)
+		$(use_enable memlimit smallmem)
+		$(use_with png)
+		$(use_with xml libxml2)
+	)
+
+	if use backend-gtk; then
```

Comment:
Please help me understand that. If there is no config switch for backend-gtk, why is it a use flag?

Reply:
- backend-gtk is required to pull x11-libs/gtk+:3
- The line "if use backend-gtk; then :" is a comment telling that --with-gtk is the default setting.

Reply:
But that makes backend-gtk a no-op use flag. If you just drop that flag, and put --with-gtk into `else`, then it is the default automatically and the `REQUIRED_USE` can be changed to `?? ( ... )` for the alternative backends.

Reply:
I don't know the syntax for `!backend-{fb,sdl,svgalib}` to pull `x11-libs/gtk+:3`. Can you be more specific please?

Reply:
that would be `!backend-fbcon? ( !backend-sdl? ( !backend-svga? ( !backend-X? ( x11-libs/gtk+:3 ) ) ) )` in rdepends.

Reply:
Thanks. Please take another look at the patch.